### PR TITLE
Fix typo in determine_pkg_source() docs

### LINKED ISSUE #328: Typo in determine_pkg_source() docs
`pkg_install` appears twice in the `@return` values of `determine_pkg_source()`, and `pkg_source` is missing.

https://github.com/pharmaR/riskmetric/blob/9e42e486651f5725a02362f2df1ac33a83921e30/R/pkg_ref_class.R#L295-L296

### PR DESCRIPTION
**Pull request by fh-kpikhart**
Closes #328

### CODE REVIEW

**Review by dgkf — APPROVED** [review]
Thank you for the correction! I'll wait for CI/CD to finish, but will plan to merge the changes after getting the green light.

**Review by emilliman5 — APPROVED** [review]

**Pull request merged**
